app/assets/stylesheets/mobile/login-modal.scss

```diff
@@ -1,292 +0,0 @@
-.static-login {
```

Comment:
I believe we're still using this class in the full page view.

Reply:
We have `app/assets/stylesheets/common/base/static-login.scss` that seems to cover this